src/main/java/Eliz.java

```diff
@@ -0,0 +1,125 @@
+                + tasks[taskNumInt - 1].description);
+    }
+
+    public static void unmarkATask(String line, Task[] tasks) {
+        char taskNumChar = line.charAt(line.length() - 1);
+        int taskNumInt = Character.getNumericValue(taskNumChar);
+        tasks[taskNumInt - 1].setAsNotDone();
+        System.out.println("[" + tasks[taskNumInt - 1].getTaskType() + "]" + "[ ] "
+                + tasks[taskNumInt - 1].description);
+    }
+
+    public static Task createTask(String line) {
+        Task t;
+        String[] splitTwoSections = line.split(" ",2); //0: task type, 1: rest of the words
+        String taskType = splitTwoSections[0];
+        switch (taskType) {
+        case "todo":
+            t = createTodo(taskType, splitTwoSections[1]);
+            break;
+        case "deadline":
+            t = createDeadline(taskType, splitTwoSections[1]);
+            break;
+        case "event":
+            t = createEvent(taskType, splitTwoSections[1]);
+            break;
+        default:
+            return null;
+        }
+        return t;
+    }
+
+    public static Task createTodo(String taskType, String line) {
+        Todo newTodo = new Todo(line);
+        return newTodo;
+    }
+
+    public static Task createDeadline(String taskType, String line) {
+        String newDescription = line;
+        if (line.contains("/")) {
+            String[] splitTwoSections = line.split("/", 2);
+            newDescription = splitTwoSections[0] + "(" + splitTwoSections[1] + ")";
+        }
+        Deadline newDeadline = new Deadline(newDescription);
+        return newDeadline;
+    }
+
+    public static Task createEvent(String taskType, String line) {
+        String newDescription = line;
+        if (line.contains("/")) {
+            String[] splitTwoSections = line.split("/", 2);
+            newDescription = splitTwoSections[0] + "(" + splitTwoSections[1] + ")";
+        }
+        Event newEvent = new Event(newDescription);
+        return newEvent;
+    }
+
+    public static void botIntroduction() {
+        String logo = " ____    __       __     ______ \n"
+                + "|  __|  |  |     |  |   |___  /\n"
+                + "| |__   |  |     |  |      / /  \n"
+                + "| |__|  |  |     |  |     / /  \n"
+                + "| |__   |  |__   |  |    / /___\n"
+                + "|____|  |_____|  |__|   |______|\n";
+
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Eliz");
+        System.out.println("What can I do for you?");
+    }
+
+    public static void markOrUnmark(String line, Task[] tasks, int taskCounter) {
+        if (line.contains("unmark")) {
+            System.out.println("OK, I've marked this task as not done yet:");
+            unmarkATask(line, Arrays.copyOf(tasks, taskCounter));
+        } else {
+            System.out.println("Nice! I've marked this task as done:");
+            markATask(line, Arrays.copyOf(tasks, taskCounter));
```

Comment:
Can consider creating a TaskList class to store the tasks and the relevant methods so that you don't have to keep passing the tasks around.